.github/CODEOWNERS

```
@@ -626,6 +626,7 @@ peps/pep-0744.rst  @brandtbucher
 peps/pep-0745.rst  @hugovk
 # ...
 # peps/pep-0754.rst
+peps/pep-0755.rst  @davidfstr @JelleZijlstra
```

Only those with write access can be in the CODEOWNERS:
```suggestion
peps/pep-0755.rst  @JelleZijlstra
```